.gitignore

```diff
@@ -57,3 +57,4 @@ libpython_path.txt
 .vscode
 _build
 *.bin
+assets/
```

Comment:
Thanks for contributing! Everything looks pretty good to me except `assets`.
Actually, the folder `/assets/` is removed a few days (or weeks?) ago, since we just no longer need it. Please simply remove the folder (submodule) from your `taichi` repo and `.gitignore`. Thanks! :-)